plugins/parodos/src/components/projectOverview/ProjectOverview.tsx

```diff
@@ -68,18 +68,22 @@ export const ProjectOverviewPage = (): JSX.Element => {
 
   React.useEffect(() => {
```

Comment:
I think trying to keep all this in sync will be problematic.

Could we not simplify it by removing `useEffect` and `setFilteredProjects` to something like:

```ts
  const filteredProjects = useMemo(() => {
    if(projectFilter === 'all-projects') {
      return allProjects;
    }

    return allProjects.filter(project => project.status === projectFilter);
  }, [allProjects, projectFilter]);
```

Reply:
Right, it makes sense.

However, let's keep the sorting.